.eleventy.js

```diff
@@ -28,6 +28,58 @@ module.exports = function(eleventyConfig) {
   eleventyConfig.addWatchTarget("./css/");
   eleventyConfig.addWatchTarget("./js/");
 
+    // Customize Markdown library and settings:
+  let markdownLibrary = markdownIt({
```

Comment:
I moved this earlier up the file because some of the shortcodes need this in their own code, so why declare it at the bottom.